docker-hub/onboard-business.md

```diff
@@ -46,32 +46,37 @@ After completing the steps from the welcome email, verify that your organization
 1. Go to [Billing Details](https://hub.docker.com/billing){: target="_blank" rel="noopener" class="_"} and then select on your organization's name.
 2. Under **Plan**, view your subscription. If you organization has a Docker Business subscription, you will see **Docker Business**.
 
-## Step 3: Add members
+## Step 3: Invite owners
 
-Now that you have a Docker Business organization, it's time to start adding members. Add members to your organization by using one of the following methods:
+Now that you have a Docker Business organization, it's time to start adding owners to help you set up and manage your organization. Owners can add or remove members, and configure Single Sign-on as well as other security settings.
 
-   * **Single Sign-on**: Automatically provision and add members when they sign in to Docker through your identity provider. For details, see [Configure Single Sign-on](../single-sign-on/index.md){: target="_blank" rel="noopener" class="_"}.
-   * **System for Cross-domain Identity Management**: Automatically provision, add, and de-provision members from your identity provider. For details, see [System for Cross-domain Identity Management](../docker-hub/scim.md){: target="_blank" rel="noopener" class="_"}.
-   * **Invite members**: Invite members using their Docker ID or email address. For details, see [Invite members](../docker-hub/members.md/#invite-members){: target="_blank" rel="noopener" class="_"}.
-      > **Note**
-      >
-      > Before you invite members, you must create at least one team. For details, see [Create a team](../docker-hub/orgs.md/#create-a-team){: target="_blank" rel="noopener" class="_"}.
+To add an owner, invite a user to the **owners** team. For more details, see [Invite members](../docker-hub/members.md/#invite-members){: target="_blank" rel="noopener" class="_"}.
 
+## Step 4: Invite members
 
-## Step 4: Enforce sign-in for Docker Desktop
+Add members to your organization using a CSV file, or by entering their email addresses. For more details, see [Invite members](../docker-hub/members.md/#invite-members){: target="_blank" rel="noopener" class="_"}
+
+## Step 5: Manage members with SSO and SCIM
+
+Automate adding members to your organization using the following:
+   - Single Sign-on: Automatically provision and add members when they sign in to Docker through your identity provider. For details, see [Single Sign-on overview](../single-sign-on/index.md).
```

Comment:
Should we say Docker Hub instead of Docker?

Reply:
Thanks. Updated in the commit below.